notebooks/community/model_garden/model_garden_movinet_clip_classification.ipynb

```diff
@@ -0,0 +1,871 @@
+{
```

Comment:
<p>Training Dockers -&gt; training docker images</p><p>Docker -&gt; docker (lowercase when referring to the image, aka not the product)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>